packages/wallet-sdk/src/CoinbaseWalletProvider.test.ts

```diff
@@ -72,7 +71,11 @@ describe('signer configuration', () => {
     (fetchSignerType as jest.Mock).mockRejectedValue(error);
 
     const provider = createProvider();
-    await expect(provider.request({ method: 'eth_requestAccounts' })).rejects.toThrow(error);
+    await expect(provider.request({ method: 'eth_requestAccounts' })).rejects.toMatchObject(
+      expect.objectContaining({
+        message: error.message,
+      })
+    );
```

Comment:
we should have thrown correctly serialized errors.
these tests were not correct.
fixed.